cmd/signozschemamigrator/schema_migrator/custom_retention_logs_migrations.go

```diff
@@ -0,0 +1,689 @@
+package schemamigrator
+
+var (
+	CustomRetentionLogsMigrations = []SchemaMigrationRecord{
+		{
+			MigrationID: 1,
+			UpItems: []Operation{
+				CreateTableOperation{
+					Database: "signoz_logs",
```

Comment:
these table are not required i.e logs and distributed_logs.

Reply:
If you are taking about the MV's "attribute_keys_bool_final_mv" , we can remove them as well.Check `logs_migrations.go` where they are getting dropped and recreated.

Reply:
We can remove these migrations from the v2 squashed migration @eKuG 

1,2,5,6,7,8,11,12,13, 18, 19

this will clean it up.